cli/bench/deno_common.js

```diff
@@ -6,7 +6,11 @@ Deno.bench("date_now", { n: 5e5 }, () => {
 });
 
 // Void ops measure op-overhead
-Deno.bench("op_void_sync", { n: 1e7 }, () => Deno.core.ops.op_void_sync());
+Deno.bench(
+  "op_void_sync",
+  { n: 1e7 },
+  () => Deno.core.unwrapOpResult(Deno.core.ops.op_void_sync()),
```

Comment:
`op_void_sync` is infallible. Is this to measure the conditional unwrap?

Reply:
This is indeed meant to measure the unwrap together with the op call itself. Since only infallible ops that don't take parameters are truly infallible, it stands to reason that the unwrapping is an integral part of all ops and should be measured here.